app/views/admin/redirections/index.html.erb

```diff
@@ -1,36 +1,62 @@
-<div class="header-row">
-  <div class="redirection">
-    <div>Slug</div>
-    <div class="url">URL</div>
-    <div class="original-url">Original URL</div>
-    <div class="edit"></div>
-    <div class="block-button"></div>
-    <div class="unlink-button"></div>
-  </div>
-</div>
+<h2>Sort by</h2>
+<section class="sort">
+  <%= link_to(
+    "Ring order",
+    admin_redirections_path(sort_key: "next_id", sort_dir: "desc")
+  ) %>
+  <%= link_to(
+    "Most recently seen first",
```

Comment:
Hmm, how about "Most recently seen" here and "Least recently seen" below? Sorry, this is nitpicky.